lightning_client.go

```diff
@@ -1310,6 +1323,15 @@ type InvoiceHtlc struct {
 	ResolveTime time.Time
 }
 
+// PendingHtlc represents a HTLC that is currently pending on some channel.
+type PendingHtlc struct {
+	// Incoming is true if the HTLC is incoming.
+	Incoming bool
+
+	// Amount is the amount in millisatoshis that this HTLC represents.
+	Amount btcutil.Amount
```

Comment:
Should use `lnwire.MilliSatoshi` if it's in fact millisatoshis.

Reply:
It's in satoshis though

Reply:
Okay, then fixing the comment should be enough.

Reply:
Sorry the comment was copy pasted. Fixed now.